tests/test_type_utils.py

```diff
@@ -188,6 +194,12 @@ def test_get_type_as_string(type_):
         pytest.fail(f"test get_type_as_string raised: {e}")
 
 
+def test_type_as_string_with_annotated_type():
```

Comment:
If you feel like doing more testing work we should really test the string output of these, not just that they fail. But that's out of scope.